# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/js.msg
# Commit: 3c0037ba793b
# Full Hash: 3c0037ba793b7d393449036987767a44423bf346
# Author: Jason Orendorff <jorendorff@mozilla.com>
# Date: 2018-02-24 10:13:47
# Regressor Bug: 1440431
# File Overlap Count: 1
# Description:
#   Bug 1440431 - Part 1: Move ValueToScript into the engine. r=nbp.
# ==============================================================================

diff -r 37f8ff509e9d -r 3c0037ba793b js/src/js.msg
--- a/js/src/js.msg	Fri Feb 23 09:01:08 2018 +0700
+++ b/js/src/js.msg	Thu Feb 22 12:31:01 2018 -0600
@@ -482,6 +482,9 @@
 MSG_DEF(JSMSG_DEBUG_PROMISE_NOT_REJECTED, 0, JSEXN_TYPEERR, "Promise hasn't been rejected")
 MSG_DEF(JSMSG_DEBUG_NO_BINARY_SOURCE,  0, JSEXN_ERR, "WebAssembly binary source is not available")
 
+// Testing functions
+MSG_DEF(JSMSG_TESTING_SCRIPTS_ONLY, 0, JSEXN_TYPEERR, "only works on scripts")
+
 // Tracelogger
 MSG_DEF(JSMSG_TRACELOGGER_ENABLE_FAIL, 1, JSEXN_ERR, "enabling tracelogger failed: {0}")
 